src/rpcblockchain.cpp

```diff
@@ -3270,30 +3300,36 @@ UniValue GetJSONVersionReport()
 {
     UniValue results(UniValue::VARR);
     //Returns a report of the GRC Version staking blocks over the last 100 blocks
-    std::string neural_ver = "";
     std::string report = "Version, Popularity\n";
     std::string row = "";
     double pct = 0;
     UniValue entry(UniValue::VOBJ);
     entry.pushKV("Version","Popularity,Percent %");
-
+    
```

Comment:
whitespace error
Just axe this code.

Reply:
Happy too :) I'll double check the dependencies.